terraform/alerting/alerts.tf

```diff
@@ -162,7 +163,7 @@ resource "google_monitoring_alert_policy" "HumanDecryptedValue" {
       duration = "60s"
```

Comment:
You probably need to set the duration to zero to be alerted as soon as it happens. The current config will only fire after the current condition has been sustained for 60 seconds.